### ISSUE TITLE
Error during SonarQube Scanner execution

### ISSUE THREAD

**Issue by gustavodebiasi** [gabrie-allaigre/sonar-gitlab-plugin#229]
Hello!
First of all, congrats for this package. As soon as I get it working it will be really nice to have this analisys.

I'm having an error of SonarQube, I can't figure it out why.
I'm using SonarQube and gitlab-plugin, and I'm getting this log.

Does somebody know how to fix it? Thank you so much.

```
16:48:59.501 INFO: Sensor XML Sensor [xml] (done) | time=372ms
16:48:59.501 INFO: Sensor PHP sensor [php]
16:48:59.502 INFO: 13/13 source files have been analyzed
16:48:59.923 INFO: 167 source files to be analyzed
16:49:02.862 INFO: 167/167 source files have been analyzed
16:49:02.871 INFO: No PHPUnit test report provided (see 'sonar.php.tests.reportPath' property)
16:49:02.874 INFO: No PHPUnit coverage reports provided (see 'sonar.php.coverage.reportPaths' property)
16:49:02.875 INFO: Sensor PHP sensor [php] (done) | time=3374ms
16:49:02.875 INFO: Sensor Analyzer for "php.ini" files [php]
16:49:02.881 INFO: Sensor Analyzer for "php.ini" files [php] (done) | time=6ms
16:49:02.887 INFO: ------------- Run sensors on project
16:49:02.898 DEBUG: Sensors : 
16:49:02.906 INFO: No SCM system was detected. You can use the 'sonar.scm.provider' property to explicitly specify it.
16:49:02.909 INFO: Performing issue tracking
16:49:03.123 INFO: 591/591 components tracked
16:49:03.126 INFO: Load server rules
16:49:03.450 INFO: ------------------------------------------------------------------------
16:49:03.450 INFO: EXECUTION FAILURE
16:49:03.450 INFO: ------------------------------------------------------------------------
16:49:03.450 INFO: Total time: 7:11.151s
16:49:03.592 INFO: Final Memory: 18M/361M
16:49:03.592 INFO: ------------------------------------------------------------------------
16:49:03.592 ERROR: Error during SonarQube Scanner execution
java.lang.IllegalStateException: Unable to load component class org.sonar.scanner.scan.report.JSONReport
	at org.sonar.core.platform.ComponentContainer$ExtendedDefaultPicoContainer.getComponent(ComponentContainer.java:65)
	at org.picocontainer.DefaultPicoContainer.getComponent(DefaultPicoContainer.java:678)
	at org.sonar.core.platform.ComponentContainer.getComponentByType(ComponentContainer.java:281)
	at org.sonar.scanner.scan.ProjectScanContainer.doAfterStart(ProjectScanContainer.java:369)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:136)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:122)
	at org.sonar.scanner.bootstrap.GlobalContainer.doAfterStart(GlobalContainer.java:126)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:136)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:122)
	at org.sonar.batch.bootstrapper.Batch.doExecute(Batch.java:73)
	at org.sonar.batch.bootstrapper.Batch.execute(Batch.java:67)
	at org.sonarsource.scanner.api.internal.batch.BatchIsolatedLauncher.execute(BatchIsolatedLauncher.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.sonarsource.scanner.api.internal.IsolatedLauncherProxy.invoke(IsolatedLauncherProxy.java:60)
	at com.sun.proxy.$Proxy0.execute(Unknown Source)
	at org.sonarsource.scanner.api.EmbeddedScanner.doExecute(EmbeddedScanner.java:185)
	at org.sonarsource.scanner.api.EmbeddedScanner.execute(EmbeddedScanner.java:137)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:111)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:75)
	at org.sonarsource.scanner.cli.Main.main(Main.java:61)
```

**Comment by gabrie-allaigre** [comment]
Hi,
SonarQube version please ?

**Comment by gustavodebiasi** [comment]
> Hi,
> SonarQube version please ?

I'm sorry. SonarQube Version: 7.7
Sonar Gitlab plugin: 4.0

**Comment by gustavodebiasi** [comment]
Installing version 4.1 of gitlab plugin solved the problem.
I'm sorry.
